backend/tests/schemas/test_generation.py

```
@@ -294,6 +294,51 @@ def test_optional_fields_can_be_none(self):
+        from app.schemas.generation import VideoGenerationConfig
+
+        config = VideoGenerationConfig(
+            ratio="adaptive",
+            duration=15,
+            resolution="1080p",
+            capabilities={
+                "resolutions": [
+                    {
+                        "label": "2K",
+                        "value": "1440p",
+                        "tooltip": "2560 × 1440",
```

Ruff (0.16.1)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)